pydatastructs/linear_data_structures/linked_lists.py

```diff
@@ -111,6 +111,8 @@ def insert_after(self, prev_node, data):
                                  links=['next', 'prev'],
                                  addrs=[None, None])
         new_node.next = prev_node.next
+        if prev_node.next is not None:
+            prev_node.next.prev = new_node
```

Comment:
Thanks for noticing this. Can you please provide an example where the need for this arises?

Reply:
If we **insert_at**, **insert_after**, **insert_before** twice at the same position then the **prev** would be pointing to the wrong node.
**Example**
[1, 5, 10, 4, 0, 9]

dll.insert_after(dll[2],7)
[1, 5, 10, 7, 4, 0, 9]

dll.insert_before(dll[4],6)
[1, 5, 10, 7, 4, 0, 9]

Reply:
Have you added regression tests for the same?

Reply:
yes i have added